config/software/jre-from-jdk.rb

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2013-2014 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Oracle doesn't distribute JRE builds for ARM, only JDK
+# builds. Since we do no want to ship a larger package with a
+# different layout, we just pick the 'jre' folder inside the jdk.
+# This allows us to get as close as an ARM JRE package as we can.
+name "jre-from-jdk"
+default_version "8u91"
+
+unless armhf?
```

Comment:
Let's go ahead and add a 64-bit Linux `8u91` entry (and update this to `unless _64_bit? || armhf?`). That will allow this software definition to work on the most popular target (64-bit Linux) and allow our testing to work as expected.